crates/common/src/retry.rs

```diff
@@ -83,6 +83,7 @@ impl Retry {
 
     fn handle_err(&mut self, err: Error) {
         self.retries -= 1;
-        warn!("erroneous attempt ({} tries remaining): {}", self.retries, err.root_cause());
+        let _ =
+            sh_warn!("erroneous attempt ({} tries remaining): {}", self.retries, err.root_cause());
```

Comment:
Good idea to surface this to the regular CLI output

Perhaps we could format this as `Warning: Etherscan could not detect the deployment (3 tries remaining)` as erroneous attempt is a bit ambiguous

Reply:
added in https://github.com/foundry-rs/foundry/commit/3a9b873127b3915f4ad24c9006ee6476d525344f looking better

![image](https://github.com/user-attachments/assets/f59606fb-a50c-4fb7-b866-735f20e407ca)